### PR TITLE
Moves all media-box--action props to card--action

### PR DESCRIPTION
**Pull request by eduardojmatos**
**CHANGELOG** :memo:

* Moves `media-box--action` to `card--action` (removing padding of `media-box__content` and `media-box__body`. There is not necesary to this component;
* `btn--full-card` belongs to `card` component, not `media-box`.

### CODE REVIEW

**Comment by iondrimba** [comment]
@eduardojmatos Testing it locally I ran into to a small bug.
captura de tela 2017-09-18 as 6 05 17 pm

The border radius looks oval instead of rounded

**Comment by eduardojmatos** [comment]
@iondrimba I'm testing here, and I don't see this error. My print here:

screen shot 2017-09-22 at 3 04 59 am

Did you ran this locally changing classes by inspector or in the source of Gaiden at Scorpion project?

The original source of docs, here in Gaiden, is with height and width on img tag. Even like this, the bug appears don't to happen. Can you show the source too? 🙏

**Comment by iondrimba** [comment]
@eduardojmatos This error occurs on the card component, not the media box. : )

captura de tela 2017-09-22 as 9 36 18 am

**Comment by eduardojmatos** [comment]
@iondrimba nice. When I moved `--action` to `card` component, this bug comes to `card--action` as a "gift". I'll apply a fix to this case, thanks for report! =D

File: src/scss/components/_picture.scss [hunk 1]
```diff
@@ -29,7 +29,6 @@
   &__image {
     background-color: get-color(air);
     display: block;
-    height: auto;
```
Comment (eduardojmatos) [inline src/scss/components/_picture.scss]: @iondrimba removing this `height: auto` does not affect picture images. Removing here, it's ok! :)

PS: We need use width/height always in all images with this removal. This is a best practice, indeed.

File: docs/demo/gaiden-css/components/card/action.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        <button class="btn btn--regular btn--full-card space-element-large">pedir orçamento</button>
+      </article>
+    </div>
+
+    <div class="col-small-6">
+      <article class="card card--profile card--action media-box media-box--action">
+        <header class="media-box__content">
+          <a class="media-box__profile" href="#">
+            <picture class="media-box__picture picture picture--rounded">
+              <img class="picture__image" src="/demo/gaiden-css/images/services/thumbnail__photo.png" alt="Exemple image" width="38" height="38">
```
[the remark this answers is not in the document]
Comment (eduardojmatos) [reply docs/demo/gaiden-css/components/card/action.html]: Sim, é verdade... Nesse caso, a imagem vai precisar sempre ter `width` e `height` por conta do rounded 😭

**Pull request merged**